app/controllers/api/v1/UsersController.js

```diff
@@ -17,6 +18,7 @@ exports.addController = function(app) {
   UsersController.create = function(req, res) {
     var newUser = new models.User({
       username: req.body.username,
+      email: _.isUndefined(req.body.email) ? "" : req.body.email,
```

Comment:
Let's try extracting this conditional to getter in model defineProperty. This will make controller slimmer.